extension.ts

```diff
@@ -391,6 +392,19 @@ export async function activate(context: vscode.ExtensionContext) {
     toggleExtension(configuration.disableExtension, compositionState);
   });
 
+  registerCommand(context, 'vim.vimTutor', async () => {
+    const textByLine = readFileSync('./src/static/vimtutor.txt').toString();
```

Comment:
I'm surprised this sort of pathing would work, as we'd be referencing a file in the extension directory.

Reply:
I'll check it out tomorrow, but how would you suggest me to add this path?

Reply:
It currently works.